packages/web-components/fast-foundation/src/radio/radio.pw.spec.ts

```diff
@@ -90,7 +90,7 @@ test.describe("Radio", () => {
             `;
         });
 
-        await expect(element).toHaveAttribute("tabindex", "");
+        await expect(element).not.toHaveAttribute("tabindex", "");
```

Comment:
Playwright's `toHaveAttribute` doesn't properly handle the difference between a present attribute with a blank value (`tabindex=""`) versus the non-existence of an attribute. This would be better expressed with `hasAttribute`:

```ts
await expect(element).not.hasAttribute("tabindex");
```